README.md

```diff
@@ -43,8 +43,10 @@ pip install -e .
 # If using macOS High Sierra or higher, run this before run setup, to allow multithreading
 # export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
 
-# Download required data files.
-nlg-eval --setup
+# Download required data and external code files.
+# If you don't like the default (~/.cache/nlgeval), specify a path instead of "default"
+# The download path is stored in ~/.config/nlgeval/rc.json
+nlg-eval --setup default
```

Comment:
Is there any way that we can just not require passing `default` so that `nlg-eval --setup` can still work?

Reply:
(This would fix the Travis build and other people that have scripts - I know they shouldn't depend directly on master/latest but it happens)

Reply:
I was wondering about the same thing, but it gets tricky with eager option evaluation :/

Reply:
i think there are two options:
- do manual checking whether references/hypothesis are provided and instead of relying on click to do it. This would result in an odd help message
- use two subcommands (setup + evaluate) and default to the latter if none are supplied. Not sure if that works, i'll test it.

Reply:
I went for the second option.